<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>170</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Loadshedding App by AmRit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Group 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group 7</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Sunday</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Monday</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Thursday</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Friday</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Saturday</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Morning</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Evening</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>85</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh </string>
    </property>
   </widget>
   <widget class="QLabel" name="refresh_at">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>339</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Refreshed yet</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>323</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
